src/amp-story-player/amp-story-player-impl.js

```diff
@@ -161,6 +161,7 @@ export class AmpStoryPlayer {
   attachCallbacksToElement_() {
     this.element_.load = this.load.bind(this);
     this.element_.show = this.show.bind(this);
+    this.element_.addStories = this.addStories.bind(this);
```

Comment:
Should it be just `add()`?

Reply:
True! I am trying to figure out where I got confused about the name not being "add". I have no idea.

Anyways, I fixed this on the latest commit!